### PR TITLE
Add information about the AUR package to README.md

### PR DESCRIPTION
**Pull request by eqrx**
**What this PR does / why we need it**:
Add information about the AUR package to README.md

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
None

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,15 @@ Support for packages managers is still work in progress and expected
 to be finished for one of the upcoming release. For details about the
 progress follow the [issue #471][17]
 
+#### Archlinux
+
+We have a package in the AUR [here](https://aur.archlinux.org/packages/kubeone).
+Use your favorite methode to build it on your system, for example by using
```
Comment (xmudrii) [inline README.md]: s/methode/method I think

File: README.md [hunk 2]
```diff
@@ -96,6 +96,15 @@ Support for packages managers is still work in progress and expected
 to be finished for one of the upcoming release. For details about the
 progress follow the [issue #471][17]
 
+#### Archlinux
```
Comment (xmudrii) [inline README.md]: Very tiny nit: s/Archlinux/Arch Linux (I think this is their official naming or at least what they use on the website)

File: README.md [hunk 3]
```diff
@@ -96,6 +96,15 @@ Support for packages managers is still work in progress and expected
...
+
+We have a package in the AUR [here](https://aur.archlinux.org/packages/kubeone).
```
Comment (xmudrii) [inline README.md]:
```suggestion
We have the [`kubeone` package in the AUR](https://aur.archlinux.org/packages/kubeone).
```

File: README.md (hunk 2 above)
Comment (xmudrii) [reply README.md]:
```suggestion
#### Arch Linux
```

File: README.md (hunk 1 above)
Comment (xmudrii) [inline README.md]:
```suggestion
Use your favorite method to build it on your system, for example by using
```

File: README.md [hunk 4]
```diff
@@ -96,6 +96,15 @@ Support for packages managers is still work in progress and expected
...
+Use your favorite methode to build it on your system, for example by using
+aurutils:
```
Comment (xmudrii) [inline README.md]:
```suggestion
`aurutils`:
```

**Comment by eqrx** [comment]
Corrected :) Thank you

**Pull request merged**